### ISSUE TITLE
Upgrade required to work for python 3

### ISSUE THREAD

**Issue by covuworie** [slanglab/phrasemachine#9]
When running on Windows with Python 3 calling the get_phrases method fails with the following stack trace:

```
C:\WinPython-64bit-3.5.2.3Qt5\python-3.5.2.amd64\lib\site-packages\phrasemachine\phrasemachine.py in get_phrases(text, tokens, postags, tagger, grammar, regex, minlen, maxlen, output)
    256     ## try to get values for both 'postags' and 'tokens', parallel lists of strings
    257     if postags is None:
--> 258         if isinstance(tagger, (str,unicode)):
    259             assert tagger in TAGGER_NAMES, "We don't support tagger %s" % tagger
    260             tagger = TAGGER_NAMES[tagger]()

NameError: name 'unicode' is not defined
```
Apparently `unicode` is not defined in python 3. Not sure if there are any other issues also.

The following [question](http://stackoverflow.com/questions/4182603/python-how-to-convert-a-string-to-utf-8) may help @brendano.

**Comment by pnulty** [comment]
As a workaround, I was able to make this work in python3 on linux by running the 2to3 script on all the .py files in the module directory.

**Comment by zhiyzuo** [comment]
+1.

Error on my machine:
```
~ $ ipython
Python 3.6.3 (default, Oct  4 2017, 06:09:15)
Type 'copyright', 'credits' or 'license' for more information
IPython 6.2.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import phrasemachine
   ...: text = "Barack Obama supports expanding social security."
   ...:

In [2]: phrasemachine.get_phrases(text)
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-2-c036f72ff88c> in <module>()
----> 1 phrasemachine.get_phrases(text)

AttributeError: module 'phrasemachine' has no attribute 'get_phrases'
```

**Comment by ZhCoding** [comment]
The same problem in my computer.
Operation System:Windows 10 64bit
Version of Python:3.5.2
>>> import phrasemachine
>>> text = "Barack Obama supports expanding social security."
>>> print (phrasemachine.get_phrases(text))
Traceback (most recent call last):
  File "<pyshell#4>", line 1, in <module>
    print (phrasemachine.get_phrases(text))
AttributeError: module 'phrasemachine' has no attribute 'get_phrases'

**Comment by doug-friedman** [comment]
This is still an issue I believe.
OS: macOS 10.14.2 (18C54)
Kernel Version: Darwin 18.2.0

>>> phrasemachine.get_phrases(text)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'phrasemachine' has no attribute 'get_phrases'

**Comment by AbeHandler** [comment]
Hi all. Thanks to a PR from @JasonKessler I think this should be resolved w/ the latest version of the package (1.0.4) now up on pip. 

Try running:

```
$ pip install -U phrasemachine
Collecting phrasemachine
  Using cached 
...
[stuff is printed here]
...
Successfully built phrasemachine
Installing collected packages: phrasemachine
Successfully installed **phrasemachine-1.0.4**    # note version number.

$ python --version 
Python 3.6.2

$ python 
>>> import phrasemachine
>>> text = "Barack Obama supports expanding social security."
>>> phrasemachine.get_phrases(text)
>>> {'num_tokens': 7, 'counts': Counter({'barack obama': 1, 'social security': 1})}
```

With these steps I am not seeing the errors referenced above and so am closing this issue. If someone could confirm the latest version of phrasemachine actually solves this for them, that would be helpful. Alternately, if you are still seeing the error let me know and I can try to help track down what is going on.

**Comment by pnulty** [comment]
I can confirm that this fixes this issue for me, Python 3.7.2. Thanks.
